src/components/DownloadTrajectoryMenu/index.tsx

```diff
@@ -24,12 +17,10 @@ const DownloadTrajectoryMenu = ({
     simulariumFile,
     isNetworkedFile,
 }: DownloadTrajectoryMenuProps): JSX.Element => {
-    const fileIsLoaded = () => !!simulariumFile.name;
+    const fileName = simulariumFile.name;
+    const isDisabled = !fileName || isBuffering;
```

Comment:
`fileIsLoaded` just did a boolean conversion on the name, and then it was required as an argument in `downloadFile` might as well just save it once and it serves both purposes